support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -130,26 +130,25 @@ function getIsRemoteFromAcquisitionData(): boolean {
 	}
 }
 
-function getTestFromAcquisitionData(): AcquisitionABTest | null | undefined {
+function getTestFromAcquisitionData(): AcquisitionABTest[] | undefined {
 	const acquisitionDataParam = getQueryParameter('acquisitionData');
 
 	if (!acquisitionDataParam) {
-		return null;
+		return undefined;
 	}
 
 	try {
 		const acquisitionData = JSON.parse(acquisitionDataParam) as {
+			abTests?: AcquisitionABTest[];
 			abTest?: AcquisitionABTest;
 		};
 
-		if (acquisitionData.abTest?.variant) {
-			return acquisitionData.abTest;
-		}
```

Comment:
We were effectively doing a bit of validation here on the shape of the object coming from the URL. Do we want to have that validation on the whole array of tests? If so I think it might be worth using a proper validation library instead of rolling our own (e.g [zod](https://github.com/colinhacks/zod))